src/main/java/seedu/address/model/Model.java

```diff
@@ -74,6 +77,12 @@ public interface Model {
      */
     void addPerson(Person person);
 
+    /**
+     * Inserts the given person at the specified index.
+     * {@code person} must not already exist in the address book.
```

Comment:
Considering only undo currently uses insertPerson maybe the Javadoc comment should specify this or the name should reflect this so that it can be better understood why this command could be used.

Reply:
No, I don't think that is necessary. 
insertPerson is a general method of the model so I don't see a need to mention how it is being used in the method description.